cv.typ.schema.json

```diff
@@ -1,1854 +1,540 @@
-                {
-                    "category": "BI \u0026 Data Analytics",
-                    "skills": [
-                        "Microsoft Excel",
-                        "Microsoft Power BI",
-                        "Jupyter Notebook",
-                        "Numpy",
-                        "Pandas",
-                        "Plotly",
-                        "Altair"
-                    ]
-                }
+            "format": "uri"
+          },
+          "position": {
+            "type": "string"
+          },
+          "startDate": {
+            "type": "string",
+            "format": "date"
+          },
+          "endDate": {
+            "type": "string"
+          },
+          "highlights": {
+            "type": [
+              "array",
+              "null"
             ],
-            "languages": [
-                {
-                    "language": "English",
-                    "fluency": "Native speaker"
-                },
-                {
-                    "language": "German",
-                    "fluency": "Native speaker"
-                },
-                {
-                    "language": "Japanese",
-                    "fluency": "Basic proficiency"
-                }
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "awards": {
+      "type": [
+        "array",
+        "null"
+      ],
+      "default": [],
+      "items": {
+        "type": "object",
+        "required": [
+          "title",
+          "date",
+          "issuer"
+        ],
+        "properties": {
+          "title": {
+            "type": "string"
+          },
+          "location": {
+            "type": "string"
+          },
+          "date": {
+            "type": "string",
+            "format": "date"
+          },
+          "issuer": {
+            "type": "string"
+          },
+          "url": {
+            "type": [
+              "string",
+              "null"
             ],
-            "interests": [
-                "Video Games",
-                "Anime",
-                "Reading Non-fiction",
-                "Classical and Original Soundtrack Music"
+            "format": "uri"
+          },
+          "highlights": {
+            "type": [
+              "null",
+              "array"
             ],
-            "references": [
-                {
-                    "name": "Dr. Jane Austin",
-                    "reference": "John was a great student. He was always eager to learn new things and was very passionate about his studies. As his mentor, I am proud to say that he was also a great leader and was able to lead his team to victory in the 2020 SCG Bangkok Business Challenge Global Competition.",
-                    "url": "https://janeaustin.com"
-                }
-            ]
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
```

Comment:
Could you remove this extra bracket here please and move it to the end of the file? It closes the properties field too early and makes the subsequent fields like certificates be in the top-level.